dockerStart.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+#
+# Little startup script fetches data dictionary, and runs the relay
```

Comment:
don't you also need to update the dockerfile to run `dockerStart.sh` instead of `CMD [ "./auspice.js", "view", "--datasetDir", "data" ]`?

Reply:
no, dockerStart.sh should be called from kubernetes only. In the dockerfile. It is still same